macros/Cigar.init.mac

```diff
@@ -0,0 +1,26 @@
+### --------------------------------------------------------
+### This init macro simulates the response of a SensL SiPM DICE
+### board in the black box.
+### --------------------------------------------------------
+
+/control/execute macros/physics/DefaultPhysicsList.mac
```

Comment:
If you use the default physics, do not repeat the 13-18 lines. Make sure you actually want to use the default list, otherwise, just remove this line.